config/adapters_config.yaml

```diff
@@ -554,4 +554,32 @@ dbsuper_super_enhancer_regulates_gene:
 
   outdir: dbsuper
   nodes: False
+  edges: True
+
+enhancer_atlas_enhancer:
+  adapter:
+    module: biocypher_metta.adapters.enhancer_atlas_adapter
+    cls: EnhancerAtlasAdapter
+    args:
+      enhancer_filepath: /mnt/hdd_2/abdu/biocypher_data/enhancer_atlas/enhancer.bed.gz
+      enhancer_gene_filepath: /mnt/hdd_2/abdu/biocypher_data/enhancer_atlas/enhancer_gene
+      tissue_to_ontology_filepath: ./aux_files/enhancer_atlas_tissues_to_ontology.pkl
```

Comment:
where is the `label` argument here?

Reply:
if label isn't present in the adapter config the default `enhancer` will be set in the adapter's constructor